src/jit/emitxarch.cpp

```diff
@@ -4797,6 +4895,18 @@ void emitter::emitIns_AX_R(instruction ins, emitAttr attr, regNumber ireg, regNu
 }
 
 #if FEATURE_HW_INTRINSICS
+void emitter::emitIns_SIMD_R_R(instruction ins, regNumber reg1, regNumber reg2, var_types simdtype)
+{
+    emitIns_R_R(ins, emitTypeSize(simdtype), reg1, reg2);
+}
+
+void emitter::emitIns_SIMD_R_R_I(instruction ins, regNumber reg1, regNumber reg2, ssize_t iVal, var_types simdtype)
+{
+#pragma warning(disable : 4244)
+    emitIns_R_R_I(ins, emitTypeSize(simdtype), reg1, reg2, iVal);
```

Comment:
Same here, which instructions hit this case?

Reply:
Two oprend legacy SSE with third operand in the form of immediate value. Not hooked up yet - see `R_R` case above for explanation.

Reply:
I don't think we want any codepaths specific to legacy instructions, as per my comment above.

CC. @fiigii

Reply:
Agree with @tannergooding.